### PR TITLE
Refactor icons

### PR DESCRIPTION
**Pull request by ashleysyg**

### CODE REVIEW

File: components/base/DTableOfContents.vue [hunk 1]
```diff
@@ -12,10 +12,11 @@
           :class="[textColor]"
           tabindex="0"
         >
-          <span v-if="entry.fa" class="icon">
-            <i
-              :class="['d-toc-icon', entry.fa.prefix, `fa-${entry.fa.icon}`]"
-            />
+          <span v-if="entry.icon" class="icon">
+            <!-- <i
+              :class="['d-toc-icon', entry.mdi.prefix, `mdi-${entry.mdi.icon}`]"
```
Comment (mcmcgrath13) [inline components/base/DTableOfContents.vue:17]: delete commented out code

File: components/blocks/DirsToCardSections.vue [hunk 2]
```diff
@@ -13,10 +13,10 @@
         <template>
           <h2
             class="section-title title has-text-dark"
-            :aria-label="'icon of' + category.fa.icon"
+            :aria-label="'icon of' + category.mdi.icon"
```
Comment (mcmcgrath13) [inline components/blocks/DirsToCardSections.vue:16]: I think we need a space after `icon of`

File: nuxt.config.js [hunk 3]
```diff
@@ -30,6 +30,11 @@ export default {
         integrity:
           'sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ',
         crossorigin: 'anonymous'
+      },
```
Comment (mcmcgrath13) [inline nuxt.config.js:33]: remove the fontawesome stylesheet

File: package.json [hunk 4]
```diff
@@ -23,6 +23,7 @@
     "prepare": "husky install"
   },
   "dependencies": {
+    "@mdi/font": "^5.9.55",
```
Comment (mcmcgrath13) [inline package.json:26]: if we're using the stylesheet link, is the dependency needed? Same with "material-design-icons"

File: components/blocks/DirsToCardSections.vue (hunk 2 above)
Comment (ashleysyg) [reply components/blocks/DirsToCardSections.vue:16]: changed this in some other parts of the code as well.

File: components/base/DTableOfContents.vue (hunk 1 above)
Comment (mcmcgrath13) [reply components/base/DTableOfContents.vue]: commented out code is still there

**Review by mcmcgrath13 — APPROVED** [review]
🚀

**Pull request merged**